### PR TITLE
episodes API implementation

### PR DESCRIPTION
**Pull request by woolfg**

### CODE REVIEW

File: db_schema/schema.sql [hunk 1]
```diff
@@ -42,4 +42,22 @@ CREATE TABLE spotifyEpisodePerformance (
...
   PRIMARY KEY (account_id, episode_id)
+);
+
+DROP TABLE IF EXISTS spotifyEpisodeMetadata;
+CREATE TABLE spotifyEpisodeMetadata (
+  account_id INTEGER NOT NULL,
+  episode_id VARCHAR(128) NOT NULL,
+  ep_name VARCHAR(2048) NOT NULL,
+  ep_url VARCHAR(2048),
+  ep_artwor_url VARCHAR(2048),
```
Comment (mre) [inline db_schema/schema.sql:53]:
```suggestion
  ep_artwork_url VARCHAR(2048),
```

File: src/db/SpotifyRepository.ts [hunk 2]
```diff
@@ -1,17 +1,60 @@
...
+    async storeEpisodesMetadata(
+        accountId: number,
+        payload: SpotifyEpisodesMetadataPayload
+    ): Promise<any> {
+        const replaceStmt = `REPLACE INTO spotifyEpisodeMetadata (
+            account_id,
+            episode_id,
+            ep_name,
+            ep_url,
+            ep_artwor_url,
```
Comment (mre) [inline src/db/SpotifyRepository.ts:27]:
```suggestion
            ep_artwork_url,
```

**Review by mre — COMMENTED** [review]
Minor typo, but apart from that all good.

**Comment by woolfg** [comment]
> Minor typo, but apart from that all good.

just wanted to follow the spotify typo model. regarding `explict` a spotify dev told me "At this scale even one byte means a lot! #savetheplanet" ... do I thought I give it a try ;)

**Comment by woolfg** [comment]
schema typo fixed in https://app.planetscale.com/services-openpodcast/openpodcast/deploy-requests/4 and https://app.planetscale.com/services-openpodcast/openpodcast/deploy-requests/5

**Pull request merged**